agent/functional_tests/testdata/taskdefinitions/agent-introspection-validator/task-definition.json

```diff
@@ -0,0 +1,24 @@
+{
+  "family": "ecsftest-agent-introspection-validator",
+  "networkMode": "host",
+  "containerDefinitions": [{
+    "image": "amazon/amazon-ecs-agent-introspection-validator:make",
+    "name": "agent-introspection-validator",
+    "memory": 50,
+    "healthCheck": {
```

Comment:
does this test need to have a healthcheck?

Reply:
No, this is unnecessary, I will remove it.